integration_test.go

```diff
@@ -125,13 +125,13 @@ func testOrbslamPosition(t *testing.T, svc slam.Service, subAlgo viamorbslam3.Su
 
 // testOrbslamInternalState checks the orbslam internal state.
 func testOrbslamInternalState(t *testing.T, svc slam.Service, dataDir string) {
-	internalStateStream, err := slam.GetInternalStateFull(context.Background(), svc, "test")
+	internalState, err := slam.GetInternalStateFull(context.Background(), svc, "test")
```

Comment:
because these are methods of the slam service, integration tests will automatically be updated when RDK has changes